docs/building-and-deploying-the-sample-solution.md

```diff
@@ -0,0 +1,256 @@
+
+- Start Azure PowerShell as administrator.
+
+> **Note:** Windows Powershell must be configured to run scripts. You can do this by running the following PowerShell command before continuing:
+> 
+> `set-executionpolicy unrestricted`
+
+- Move to the folder containing the provisioning scripts.
+
+- Run the following command:
+
+	`.\Provision-All.ps1`
+
+- At the *ServiceBusNamespace* prompt, enter the name of the Service Bus namespace you created in step 1.
+
+- At the *StorageAccountName* prompt, enter the name of the storage account you created in step 1.
+
+- At the *StreamAnalyticsJobName* prompt, enter a name for the Stream Analytics job to be created (the script will create this item).
+
+-  At the *SubscriptionName* prompt, enter the name of the subscription that you are using with your Azure account (this subscription should be the owner of the Service Bus namespace and the storage account).
+
+-  In the Sign in to Windows Azure Powershell dialog box, provide the credentials for your Azure account.
+
+### Creating the Configuration File
+
+- Using File Explorer create of a copy of the file named *mysettings-template.config* in the RunFromConsole folder as *mysettings.config*
+
+- In Visual Studio, open the *mysettings.config* file in the ScenarioSimulator.ConsoleHost project. 
+
+- Using the Azure portal, find the connection string for the eventhub01 event hub in the Service Bus namespace that you created earlier.
+
+- In Visual Studio, set the value of the Simulator.EventHubConnectionString key to the connection string for the eventhub01 event hub. Append the text `;TransportType=Amqp` to the end of the string
+
+> **Note:** It is important to specify the transport type because the default protocol is not supported by Azure Event Hub and will trigger exceptions at runtime.
+
+- Rebuild the solution. It should build successfully.
+
+### Configuring Stream Analytics
+
+The provisioning script should configure Stream Analytics automatically to receive events from the event hub and write a record of each event to blob storage. Event data is written using JSON format.
+
+To verify that the configuration was successful, perform the following steps:
+
+- Using the Azure web portal, open the Stream Analytics page.
+
+- On the Stream Analytics page, verify that a Stream Analytics job has been created using the name that you specified when you ran the provisioning script (the name that you entered in response to the *StreamAnalyticsJobName* prompt).
+
+- Click the job, and then in the menu bar click Configure
+
+- On the Configuration page, verify that the storage account is set to the value that you specified when you ran the provisioning script (the name that you entered in response to the *StorageAccountName* prompt).
+
+- In the menu bar click Inputs.
+
+- On the Inputs page, verify that an input named input01 has been created and that the type of the input is Event Hub.
```

Comment:
Input02 - ReferenceData - BlobStorage is also being created. Should we add that?